### DISCUSSION TITLE
Cursor in "list items around player" needs a serious do-over.

### DISCUSSION THREAD

**Discussion by Unrepentant-Atheist** [cataclysmbn/Cataclysm-BN#3490] [General]
### Is your feature request related to a problem? Please describe.

![image](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/3938423/d7a45178-e467-4c30-a740-12625d63edf3)

Quick, no thinking, which tile is it pointing to?

### Describe the solution you'd like

The cursor needs to be way, way more distinguishable. Maybe blinking, maybe a big, fat, yellow/red X, maybe a fat, yellow/red blinking. Anything but what currently is available.

**Comment by scarf005** [comment]
thanks for the suggestion. it seems to better fit in discussions, however.

**Comment by Unrepentant-Atheist** [comment]
Never sure when to open a discussion or a feature request...sorry.
